frontend/src/components/Buttons/FilterButton.vue

```diff
@@ -1,182 +1,180 @@
 <template>
   <v-menu :offset-y="true" :close-on-content-click="false" max-width="250px">
-    <template #activator="{ on, attrs }">
+    <template #activator="{ props }">
       <v-btn
         v-if="!$vuetify.display.smAndDown"
         :disabled="disabled"
         class="ma-2"
         variant="text"
         rounded
-        v-bind="attrs"
-        v-on="on">
+        v-bind="props">
         {{ $t('filter') }}
         <v-icon end>
           <i-mdi-menu-down />
         </v-icon>
       </v-btn>
-      <v-btn
-        v-else
-        :disabled="disabled"
-        class="my-2"
-        icon
-        v-bind="attrs"
-        v-on="on">
+      <v-btn v-else :disabled="disabled" class="my-2" icon v-bind="props">
         <v-icon>
           <i-mdi-filter-variant />
         </v-icon>
       </v-btn>
     </template>
-    <v-expansion-panels accordion flat focusable class="dropdown">
+    <v-expansion-panels variant="accordion" class="dropdown">
       <v-expansion-panel :title="$t('status')">
-        <v-list dense class="filter-content">
-          <v-list-group
-            v-model="selectedStatusFilters"
-            multiple
-            @change="emitFilterChange">
+        <v-expansion-panel-text>
+          <v-list
+            select-strategy="multiple"
+            class="filter-content"
+            v-model:selected="selectedStatusFilters"
+            @update:selected="emitFilterChange">
             <template
               v-for="(status, statusIndex) in statusFilters"
               :key="`status-${statusIndex}`">
               <v-list-item :value="status.name">
```

Comment:
```suggestion
              <v-list-item :value="status.name" :title="status.label">
```
I'd rather you use the `title` attribute instead of the `<v-list-title>` element (same for the other occurrences)

Reply:
That's better indeed. Done.